packages/pointer_interceptor/pubspec.yaml

```diff
@@ -1,10 +1,10 @@
 name: pointer_interceptor
 description: A widget to prevent clicks from being swallowed by underlying HtmlElementViews on the web.
-version: 0.8.0+2
+version: 0.9.0-nullsafety.0
```

Comment:
Once https://github.com/flutter/packages/pull/289 lands this can just be `0.9.0`, so we don't have to do a second pass over this package later.